Cargo.toml

```diff
@@ -95,7 +95,7 @@ strength_reduce = "0.2"
 strum = "0.27"
 strum_macros = "0.27"
 tokio = { version = "1.44", default-features = false }
-tokio-util = "0.7.8"
+tokio-util = ">=0.7.13, <0.8.0"
```

Comment:
You can just write this as `"0.7.13"`.

Reply:
Oh, I didn't know. Thanks!

Reply:
Wait. Do I not need to also include a tilde? https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements

Reply:
No you don't, caret (the default) already does what we want here.